src/main/java/org/craftercms/studio/api/v2/repository/blob/StudioBlobStore.java

```diff
@@ -28,7 +33,7 @@
  * @since 3.1.6
  */
 public interface StudioBlobStore extends BlobStore, ContentRepository,
-        org.craftercms.studio.api.v2.repository.ContentRepository {
+        org.craftercms.studio.api.v2.repository.ContentRepository, PublishCapableContentRepository {
```

Comment:
Let's add a note in the v1 refactor about this interface "PublishCapable" so we can better refactor as needed.